services/web/server/src/simcore_service_webserver/projects/_trash_api.py

```diff
@@ -69,6 +69,7 @@ async def trash_project(
     user_id: UserID,
     project_id: ProjectID,
     force_stop_first: bool,
+    explicit: bool = True,
```

Comment:
why do we need this default in the API? are you not in control of every side of the code?

Reply:
because when I call from the folders to trash the child project, I want use the project-trash API. Therefore only the caller know whether it is an explicit or implicit trash.

Reply:
and who is the caller? it's you no? 
my problem with defaults, is that usually only the author of the code knows about them. then it's very difficult for other developers to use these functions correctly

Reply:
rmed